src/android/src/main/java/org/pgsqlite/SQLitePlugin.java

```diff
@@ -357,6 +357,7 @@ private void startDatabase(String dbname, JSONObject options, CallbackContext cb
     private SQLiteDatabase openDatabase(String dbname, String assetFilePath, int openFlags, CallbackContext cbc) throws Exception {
```

Comment:
This is where i've got stucked a little bit. While debugging i found out that org/pgsqlite/SQlitePlugin.java will be used and i wondered about both implementations. Can you provide some more information about the architecture especially of the Android-Part? The Objective-C-Part is totally clear to me :)